spec/unit/ssl/verifier_spec.rb

```diff
@@ -0,0 +1,124 @@
+#! /usr/bin/env ruby
```

Comment:
Can this spec actually be run as a script? Nearly all of our spec files can't, so I've been pulling out the shebang line to people don't try, and get confused about why it's not working.

Reply:
Ah thanks! copy/pasta mistake, let me remove that.